crates/frontend/src/components/condition_pills/types.rs

```diff
@@ -116,23 +118,21 @@ impl TryFrom<&Value> for Condition {
     }
 }
 
-impl TryFrom<&Context> for Vec<Condition> {
+impl TryFrom<&Context> for Conditions {
```

Comment:
INFO: `Context` type is no more a local type in `frontend` crate, so to implement
`impl TryFrom<&Context> for Vec<Condition> {`
need to change `Vec<Condition>` to `Conditions` so that at least one of the participants in the trait implementation becomes local